test/outputFormatters-pre-imported.formatter.json

```diff
@@ -0,0 +1,93 @@
+[
+ {
+  "fileName": "viewme.md",
+  "lineNumber": 3,
+  "ruleNames": [
+   "MD009",
+   "no-trailing-spaces"
+  ],
+  "ruleDescription": "Trailing spaces",
+  "ruleInformation": "https://github.com/DavidAnson/markdownlint/blob/v0.20.4/doc/Rules.md#md009",
```

Comment:
Please rebase onto `main` branch - "v0.20.4" should be "vX.Y.Z" now - sorry I didn't have it this way originally.